src/main/kotlin/no/nav/sykdig/nasjonal/api/NasjonalOppgaveDataFetcher.kt

```diff
@@ -48,5 +42,22 @@ class NasjonalOppgaveDataFetcher(
         return null
     }
 
+    @PostAuthorize("@oppgaveSecurityService.hasAccessToNasjonalSykmelding(#sykmeldingId, #authorization, '/dgs/nasjonal/oppgave/{sykmeldingId}')")
+    @DgsQuery(field = DgsConstants.QUERY.NasjonalFerdigstiltOppgave)
+    fun getFerdigstiltNasjonalOppgave(sykmeldingId: String, authorization: String, dfe: DataFetchingEnvironment): NasjonalOppgaveResult? {
+        val oppgave = nasjonalOppgaveService.getOppgaveBySykmeldingId(sykmeldingId, authorization)
+        if (oppgave != null) {
+            if (!oppgave.ferdigstilt) {
+                log.info("Oppgave med sykmeldingId $sykmeldingId er ikke ferdigstilt")
+                return null
```

Comment:
Kan legge til IKKE_FERDIGSTILT i `NasjonalOppgaveStatusEnum`, så kan vi vise ei melding i frontend i `NasjonalOppgaveStatus` om at den er ferdigstilt. 
```suggestion
                return NasjonalOppgaveStatus(oppgaveId, NasjonalOppgaveStatusEnum.IKKE_FERDIGSTILT)
```

Reply:
God ide